src/full/Agda/Utils/SmallSet.hs

```diff
@@ -69,7 +69,7 @@ notMember a = not . member a
 -- | Time O(n).
 isSubsetOf ::  SmallSetElement a => SmallSet a -> SmallSet a -> Bool
 isSubsetOf s t = and $ toBoolListZipWith implies s t
-  where implies a b = if a then b else True
+  where implies a b = not a || b
```

Comment:
That can't not be true!

Reply:
Maybe I'm oversleepy, but looks OK to me

Reply:
(Changed back anyway)

Reply:
I was just commenting on the fact that this would not hold in constructive logic, but of course booleans are classical so it doesn't matter anyway.

Reply:
Classical logic makes my head hurt ;-), thus I prefer the `if a then b else True` form.  My brain has mostly unlearned negation.

Reply:
(I reverted it)